contracts/gateway/AxelarAmplifierGateway.sol

```diff
@@ -53,6 +53,7 @@ contract AxelarAmplifierGateway is BaseAmplifierGateway, BaseWeightedMultisig, U
     function _setup(bytes calldata data) internal override {
         (address operator_, WeightedSigners[] memory signers) = abi.decode(data, (address, WeightedSigners[]));
 
+        // operator is an optional parameter, it can also be removed in the future to make ownership completely decentralized
```

Comment:
```suggestion
        // operator is an optional parameter. The gateway owner can set it later if needed.
        // This also simplifies setup for upgrades so if the current operator doesn't need to be changed, then it can be skipped, instead of having to specify the current operator again.
```
whether ownership is decentralized depends on what the actual owner is set to. The contract code doesn't know about this, so any comments should be with respect to the roles in the contract